bundles/org.smarthomej.binding.notificationsforfiretv/src/main/java/org/smarthomej/binding/notificationsforfiretv/internal/NotificationsForFireTVBindingConstants.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
```

Comment:
Remove this line (also in all other files). Since this is not forked but a new contribution, the openHAB contributors are not copyright holders of these files.

Reply:
done